src/subsplit_dag_storage.hpp

```diff
@@ -222,6 +204,8 @@ class GenericNeighborsView {
     return result;
   }
 
+  void Remap(const T& neighbors) { neighbors_ = neighbors; }
```

Comment:
Is this the clearest name for this method? It seems more like a `SetNeighbors`.

Reply:
Good idea.  Name changed.